src/style.css

```diff
@@ -1 +1,30 @@
 /* write your styles here */
+* {
+  width: 300px;
+  height: 100px;
+  margin: 0 auto;
+}
```

Comment:
now your body have 300px width and 100px height. Apply this styles only to the blocks

Reply:
Fixed